x-pack/test/spaces_api_integration/common/fixtures/kbn_archiver/spaces_default.ndjson

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
Note that this file only includes saved objects, not Spaces themselves. Those cannot be created or imported using the regular bulkCreate/import APIs.